.github/workflows/cargo_hack.yml

```diff
@@ -15,16 +17,16 @@ env:
 
 jobs:
   hack:
-    runs-on: ubuntu-20.04
-    name: Cargo Hack Check - Powerset
+    runs-on: [self-hosted, linux, x64, general]
+    name: Cargo Hack Check
     steps:
       - uses: actions/checkout@v2
       - run: sudo bash scripts/environment/bootstrap-ubuntu-20.04.sh
       - run: echo "$HOME/.cargo/bin" >> $GITHUB_PATH
       - run: bash scripts/environment/prepare.sh
       - run: echo "::add-matcher::.github/matchers/rust.json"
       - run: cargo install cargo-hack
-      - run: cargo hack check --feature-powerset --clean-per-run
+      - run: cargo hack check --each-feature --clean-per-run
```

Comment:
Do we need `--clean-per-run`? Looking at https://github.com/taiki-e/cargo-hack#--clean-per-run it seems like this really only effects whether we would see clippy warnings running each feature individually. Are there warnings that would only show up in that case but not in a general `cargo clippy`?

Reply:
Hmm. No, that's just me being a reflexive caller of `clean` because of bad experiences with build systems that aren't cargo. I will remove this. Betcha that'll make running this much faster too.